CMSgov/mint-app
# [EASI-2838] Help article: 2-pager meeting

# EASI-2838

## Changes and Description

- Adjusted the `RelatedArticles` component 
    - classname in the padding
    - Incorporate the `View all help articles`
- `twoPageMeeting.ts` translations
- src/views/HelpAndKnowledge/Articles/TwoPagerMeeting/index.tsx compoent 



## How to test this change

1. Navigate to `http://localhost:3005/help-and-knowledge`
2. See that new `How to have a successful 2-pager meeting` card is showing under the `Getting started` cards
3. Click through and ensure article matches the figma
4. Click through the `Recommendations` links to scroll around the article.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
